examples/sdk/reactNative/android/app/src/main/java/com/rnbtexample/MainActivity.kt

```diff
@@ -0,0 +1,22 @@
+package com.rnbtexample
```

Comment:
the directory  name and the namespace is invalid - the app name is reactNative not rnbtexample

Reply:
renamed back to `reactNative`